test/hack/cleanup/main.go

```diff
@@ -35,9 +39,15 @@ const (
 
 	karpenterProvisionerNameTag = "karpenter.sh/provisioner-name"
 	karpenterLaunchTemplateTag  = "karpenter.k8s.aws/cluster"
+	karpenterSecurityGroupTag   = "karpenter.sh/discovery"
 	githubRunURLTag             = "github.com/run-url"
 )
 
+type resources interface {
+	cleanupResources(ctx context.Context)
+	getOldResources(ctx context.Context) (ids []string, err error)
```

Comment:
```suggestion
	getExpired(ctx context.Context) (ids []string, err error)
```

Reply:
Why does this interface even need this method if it's not being called by some external thing?